patterns/2-structured/service-vs-library.md

```diff
@@ -79,13 +85,16 @@ Related to this pattern is the [30 Day Warranty](30-day-warranty.md) pattern tha
 
 Europace AG
 
+Flutter Entertainment: typical InnerSource application has a shared code "service" repository with cross-team contribution and CI pipeline to build and publish a shared release artefact. Each adopting team has a "deployment config" repository defining their own deployment. This is driven by varying regulatory requirements, service and incident management practices and infrastructure skillsets in different areas of the business.
```

Comment:
The additional material is great. One thing I would be worried about long term is whether that link will stay up and remain the same.

Reply:
Of course the link might break eventually (that's what links tend to do :)).

However I think it is better for that to happen than to make it harder than necessary for pattern readers to find this extra material that could be really useful for them. When the link breaks we can update it (or remove it).

Further if it were to motivate people to contribute to the patterns because they get a back-link to their company, that wouldn't be too bad of a trade either.

Reply:
Added the link. It's my maintained content I'm linking to, so would try to submit a correction if it was to move/change.